### PR TITLE
Performance testing

### PR DESCRIPTION
**Pull request by cxn-itichy**

### CODE REVIEW

**Review by cxn-pkovacs — APPROVED** [review]
I personally wouldn't use JMeter for testing HTTP-service, because (1) JMeter is very cumbersome to use in general (including unreadable generated configuration files) (2) JMeter is an old-timer optimized for HTTP endpoints designed to be used by WEB-browsers.

I would use [k6](https://k6.io/) instead. More modern, much easier to use.

File: README.md [hunk 1]
```diff
@@ -154,3 +154,22 @@ curl --location --request POST 'http://localhost:9990/update' \
...
+
+For testing performance a simple JMeter [test plan](jmeter/performancetest.jmx) has been created. Performance test is designed to simulate real user load (calling `POST /update`) based on this rule:
+
+Start 100 threads (simulating 100 simultaneously working users) doing:
+
+```
+while (true) {
+    Fetch new test data from smiles.csv    
+    Perform /update POST call
+    Think time 4 sec (default delay for real time plugin call set up in Design Hub)
```
Comment (cxn-astracz) [inline README.md]: think time is closer to 2 seconds, on average around 1.7s actually.

**Comment by cxn-itichy** [comment]
> I personally wouldn't use JMeter for testing HTTP-service, because (1) JMeter is very cumbersome to use in general (including unreadable generated configuration files) (2) JMeter is an old-timer optimized for HTTP endpoints designed to be used by WEB-browsers.
> 
> I would use [k6](https://k6.io/) instead. More modern, much easier to use.

Hi Peti, thank you for that info. I quickly checked their web and it seems much modern. However this tool is not free which is not the best for customers. I also noticed two things, first that JMeter is very well known - like every perf guy know and second there might be other project we use JMeter in.

File: README.md (hunk 1 above)
Comment (cxn-itichy) [reply README.md]: Logic was changed to use 2 seconds pacing time.

**Review by cxn-astracz — APPROVED** [review]

**Pull request merged**